lib/reek/context/module_context.rb

```diff
@@ -20,6 +20,11 @@ def node_instance_methods
       def descriptively_commented?
         CodeComment.new(exp.leading_comment).descriptive?
       end
+
+      def namespace_module?
```

Comment:
Could we add a short comment here giving an example of what "exp" could look like and / or what we expect it is?